datapublic/common_fields.py

```diff
@@ -152,6 +152,18 @@ def list(cls):
         FieldGroup.HEALTHCARE_CAPACITY,
     )
 
+    # HSA level Healthcare Fields
+    STAFFED_BEDS_HSA = "staffed_beds_hsa", FieldGroup.HEALTHCARE_CAPACITY
+    HOSPITAL_BEDS_IN_USE_ANY_HSA = "hospital_beds_in_use_any_hsa", FieldGroup.HEALTHCARE_CAPACITY
+    CURRENT_HOSPITALIZED_HSA = "current_hospitalized_hsa", FieldGroup.HEALTHCARE_CAPACITY
+    ICU_BEDS_HSA = "icu_beds_hsa", FieldGroup.HEALTHCARE_CAPACITY
+    CURRENT_ICU_HSA = "current_icu_hsa", FieldGroup.HEALTHCARE_CAPACITY
+    CURRENT_ICU_TOTAL_HSA = "current_icu_total_hsa", FieldGroup.HEALTHCARE_CAPACITY
+    WEEKLY_NEW_HOSPITAL_ADMISSIONS_COVID_HSA = (
```

Comment:
I don't think we need to add `NEW_HOSPITAL_ADMISSIONS_COVID` here because we won't have any county-level data for that field, but it also wouldn't hurt to add it.